Sources/SwiftInspectorVisitors/TypeDescription.swift

```diff
@@ -42,6 +42,8 @@ public enum TypeDescription: Codable, Hashable {
   indirect case dictionary(key: TypeDescription, value: TypeDescription)
   /// A tuple. e.g. (Int, String)
   indirect case tuple([TypeDescription])
+  /// A closure. e.g. (Int, Double) throws -> String
+  indirect case closure(arguments: [TypeDescription], doesThrow: Bool, returnType: TypeDescription)
```

Comment:
Instead of `doesThrow` should this be something like `keywords` so we can extend it later on? It could be something like an OptionSet, and then we could have a convenience parameter for `doesThrow`.

Reply:
I think as we add more types we can talk about simplifying, but we've got only one potential new parameter in the future that we know of so I'm hesitant to optimize just yet.